tf-modules/archive/sf_event_sqs_to_db_records.tf

```diff
@@ -166,7 +166,7 @@ resource "aws_lambda_function" "sf_event_sqs_to_db_records" {
       PdrsTable       = var.dynamo_tables.pdrs.name
       DeadLetterQueue = aws_sqs_queue.sf_event_sqs_to_db_records_dead_letter_queue.id
       databaseCredentialSecretArn = var.rds_user_access_secret_arn
-      RDS_DEPLOYMENT_CUMULUS_VERSION = "5.0.0"
+      RDS_DEPLOYMENT_CUMULUS_VERSION = "4.0.0"
```

Comment:
(I'm assuming with the pending 5.0 release this should be noted to be sorted with the merge back to the feature-rds branch)

Reply:
Yeah I guess I'll wait to merge this PR until we have master back into the feature branch. Then bump this to 6.0.0

Reply:
Having this value subject to manual control like this seems somehow bad, but I'm not sure of an alternative